### ISSUE TITLE
latitude/longitude in mysql

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mousadialo/django-profile#3]
```
current latitude/longitude fielelds have this params:
DecimalField(max_digits=8, decimal_places=6, default=-100)

It fails on Mysql (4.1) with "Data trancation" warning when when the 
default value provided(-100)

Coorect needs to be: max_digits=9
```

Original issue reported on code.google.com by `ppr.vit...@gmail.com` on 5 Mar 2008 at 1:56

**Comment by GoogleCodeExporter** [comment]
```
Totally true. This was corrected in the new version 0.2

```

Original comment by `david.ru...@gmail.com` on 5 Mar 2008 at 2:40

**Comment by GoogleCodeExporter** [comment]
Original comment by `david.ru...@gmail.com` on 5 Mar 2008 at 2:52
- Changed state: **Fixed**
